views/vote/optIn.pug

```diff
@@ -0,0 +1,20 @@
+extends ../base
+
+block variables
+  -var title = __('Register to vote')
+  -var confirmOptIn = !hasData(data, 'vote.confirmOptIn') ? '' : data.vote.confirmOptIn
+
+block content
+
+  h1 #{title}
+
+  div
+    p #{__('Would you like to register to vote in the federal election?')}
+
+  form.cra-form(method='post')
+    +radiosYesNo('confirmOptIn', 'Would you like to register to vote in the federal election?', confirmOptIn)
+
+    input#redirect(name='redirect', type='hidden', value='/vote/confirmRegistration')
```

Comment:
Usually we put the 'No' URL in here instead of the 'Yes' URL. ie, if you say No, you end up redirected to this URL.